src/utils/index.test.ts

```diff
@@ -1,4 +1,4 @@
-import { isAddress, shortenAddress } from '.'
+import { isAddress, shortenAddressStrict } from './addresses'
```

Comment:
this test file should be renamed `addresses.test.ts` as it only tests functions from that file